reactioncommerce/reaction
# chore: update reaction to use reaction-eslint-rules v2.1.0

Impact: **minor**  
Type: **chore**

Update Reaction to use the latest version of [Reaction ESLint Config](https://github.com/reactioncommerce/reaction-eslint-config), which introduces linting for React Hooks.

## Breaking changes
None

## Testing
1. See that the npm package of `@reatctioncommerce/esint-config` is `v2.1.0`.
1. See that `eslint-plugin-react-hooks` is installed along side as a `devDependency`
1. See that the app starts with no errors
1. See that there are no lint `errors` (`warnings` are OK)
